include/boost/numeric/ublas/tensor/tensor_engine.hpp

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) 2018-2020, Cem Bassoy, cem.bassoy@gmail.com
+//  Copyright (c) 2019-2020, Amit Singh, amitsingh19975@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Google and Fraunhofer IOSB, Ettlingen, Germany
+//
+
+#ifndef BOOST_UBLAS_TENSOR_ENGINE_IMPL_HPP
+#define BOOST_UBLAS_TENSOR_ENGINE_IMPL_HPP
+
+#include <boost/numeric/ublas/tensor/tensor_core.hpp>
+#include <boost/numeric/ublas/tensor/layout.hpp>
```

Comment:
Are those two required in this file?

Reply:
nope, the layout does not need to be required by tensor_engine must have been the remains of the refactoring, I will remove it.